src/index.html

```diff
@@ -11,6 +11,17 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#site" class="form form--01"
```

Comment:
```suggestion
    <form action="#site" class="form form--big"
```
this modifier is more descriptive 😄